modules/apps/remote-app/remote-app-admin-web/src/main/java/com/liferay/remote/app/admin/web/internal/portlet/CustomElementPortlet.java

```diff
@@ -104,6 +106,16 @@ public synchronized void register(BundleContext bundleContext) {
 			bundleContext.registerService(
 				ResourceBundleLoader.class,
 				locale -> _getResourceBundle(locale), properties);
+
+		properties = new Hashtable<>();
```

Comment:
You can use the HashMapDictionaryBuilder, Brian lately is preferring over new HashMap/HashTable.

Reply:
Done